### DISCUSSION TITLE
is there a tutorial for a sub/pub java app?

### DISCUSSION THREAD

**Discussion by Med-salem-Guenaya** [aws/aws-iot-device-sdk-java-v2#524] [Q&A] [answered]
Im new and just want a simple java app that sub/pub to a topic, but i cant get the samples to work for me....is there a tutorial that i can follow?

**Comment by jmklix** [comment, marked as answer]
You can follow the getting started guide on the AWS console. That will set up the policies for you and then download the certs/script to setup and run the PubSub sample. You can also let me know if you have any questions about how to run any of the samples. (you can also check out our [FAQ](https://github.com/aws/aws-iot-device-sdk-java-v2/blob/main/documents/FAQ.md))
